modules/Core/module.php

```diff
@@ -548,14 +548,16 @@ public function __construct(Language $language, Pages $pages, User $user, Naviga
         Email::addPlaceholder('[Message]', static fn(Language $viewing_language, string $email) => $viewing_language->get('emails', $email . '_message'));
         Email::addPlaceholder('[Thanks]', static fn(Language $viewing_language) => $viewing_language->get('emails', 'thanks'));
 
-        MemberListManager::getInstance()->registerListProvider(new RegisteredMembersListProvider($language));
-        MemberListManager::getInstance()->registerListProvider(new StaffMembersListProvider($language));
-
-        MemberListManager::getInstance()->registerMemberMetadataProvider(function (User $member) use ($language) {
-            return [
-                $language->get('general', 'joined') => date(DATE_FORMAT, $member->data()->joined),
-            ];
-        });
+        if (Util::isModuleEnabled('Members')) {
+            MemberListManager::getInstance()->registerListProvider(new RegisteredMembersListProvider($language));
```

Comment:
Should these providers be registered within the Members module itself?

Reply:
No, the idea here is that they belong to the Core module, the member list module doesn't have any lists of it's own. Similar to how we have `forum post count` lists that are initialized in the forum module

Reply:
The reason I ask is because it feels a bit strange to have logic depending on other modules in the core. If we are definitely happy with this approach then I'm okay with it.